frontend/src/Pages/BrowsePage/BrowseResults/BrowseResults.tsx

```diff
@@ -34,22 +35,34 @@ const BrowseResults: FC = () => {
   const [error, setError] = useState<string | null>(null);
   const [page, setPage] = useState(0);
   const [rowsPerPage, setRowsPerPage] = useState(10);
-  const [orderBy, setOrderBy] = useState<keyof JobInterface>('title');
-  const [order, setOrder] = useState<Order>('asc');
+  const [orderBy, setOrderBy] = useState<keyof JobInterface>("title");
+  const [order, setOrder] = useState<Order>("asc");
 
   useEffect(() => {
     // Use the mock data instead of fetching from the API
     const fetchJobs = async () => {
       try {
         setLoading(true);
-        // Simulate an API call delay
-        setTimeout(() => {
-          setJobs(sampleJsonResponse.jobs); // Set jobs from the mock data
-          setLoading(false);
-        }, 1000);
+        await axios({
```

Comment:
So nice to see that first connection! I had hoped to start on this tomorrow, so WTG getting that going!

Reply:
🎉🎉🎉